cmd/revoke.go

```diff
@@ -26,6 +26,10 @@ func NewRevokeCommand() cli.Command {
 		Usage:       "Revoke certificate",
 		Description: "Add certificate to the CA's CRL.",
 		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "passphrase",
```

Comment:
If this flag is omitted, will we prompt for passphrase?

Reply:
Yes - I added a manual example in the PR description to show what happens.